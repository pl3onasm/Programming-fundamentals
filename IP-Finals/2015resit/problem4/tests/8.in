78
28 60 18 17 46 21 14 61 2 15 73 54 11 40 36 74 43 66 63 70 49 37 50 48 55 4 10 23 59 24 19 41 1 26 71 35 27 58 75 8 34 33 38 39 64 7 45 20 51 67 13 68 16 30 12 77 3 25 0 31 53 9 44 5 42 56 32 47 52 69 57 29 22 62 72 65 76 6
