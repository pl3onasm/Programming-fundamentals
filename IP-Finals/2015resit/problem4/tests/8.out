0 68
11 3
20 4
35 1
69 1
76 1
